### ISSUE TITLE
NGINX - Error 404 - Plugin fails when testing

### ISSUE THREAD

**Issue by Armnol** [edgardmessias/glpi-singlesignon#21]
Hi,

 I have configured an Azure application, retrieved the secret, tenant ID and client ID. However when I test the plugin via the "Test Single Sign-on" button I get a 404 error from NGINX on the callback URL. Moreover, when I activate the plugin, the login page does not show the connection options. GLPI version is 9.5.6, singlesignon is 1.3.1 and I have fusion Inventory and GLPI Modification activated.

Thanks

**Comment by edgardmessias** [comment]
Closed in favor of #26

**Comment by jrvidotti** [comment]
@edgardmessias I think that this problem is not the same of #26.

@Armnol @h33raj 

This is caused by a misconfiguration in NGINX. In my case, I changed this:

```
        location ~ \.php$ {
                include snippets/fastcgi-php.conf;
                fastcgi_pass unix:/run/php/php8.1-fpm.sock;
        }
```

To this:

```
        location ~ \.php((/.*)|$) {
                include snippets/fastcgi-php.conf;
                fastcgi_pass unix:/run/php/php8.1-fpm.sock;
        }
```
